src/Public/Remove-CCMStaleComputer.ps1

```diff
@@ -0,0 +1,72 @@
+function Remove-CCMStaleComputer {
+    <#
+    .SYNOPSIS
+    Deletes stale computers from the CCM database.
+    
+    .DESCRIPTION
+    Deletes stale computers from the CCM database.
+    
+    .PARAMETER ComputerId
+    The id of the computer to remove.
+    
+    .PARAMETER Force
+    Don't prompt for confirmation.
+    
+    .EXAMPLE
+    Remove-CCMStaleComputer -ComputerId 1
+
+    .EXAMPLE
+    Get-CCMStaleComputer -StaleAge 30 | Remove-CCMStaleComputer -Force
+    
+    .NOTES
+    
+    #>
+    [cmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+    Param(
+        [Alias('Id')]
+        [Parameter(Mandatory, ValueFromPipeline, ValueFromPipelineByPropertyName)]
+        [Int[]]
+        $ComputerId,
```

Comment:
```suggestion
        [Parameter(Mandatory, ParameterSetName = 'Name')]
        [string[]]
        $Name,
        
        [Alias('Id')]
        [Parameter(Mandatory, ValueFromPipeline, ValueFromPipelineByPropertyName, ParameterSetName = 'Id')]
        [Int[]]
        $ComputerId = (Get-CCMComputer -Computer $Name).Id,
```

I'm always a huge fan of this lazy pattern, though strictly speaking PSSA isn't.

Reply:
Ha, this is basically how I've implemented this!